booking/README.md

```diff
@@ -41,31 +44,32 @@ The booking app uses [go-sqlite3](https://github.com/mattn/go-sqlite3) database
 $ brew install pkgconfig sqlite3
 ~~~
 
-### To install on Ubuntu:
-
-	$ sudo apt-get install sqlite3 libsqlite3-dev
-
-Once you have SQLite installed, it will be possible to run the booking app:
+### Install sqlite on Ubuntu:
+```sh
+$ sudo apt-get install sqlite3 libsqlite3-dev
+```
 
+Once SQLite is installed, it will be possible to run the booking app:
+```sh
 	$ revel run github.com/revel/samples/booking
+```
 
 ## Database / Gorp Plugin
 
-[`app/controllers/gorp.go`](https://github.com/revel/samples/blob/master/booking/app/controllers/gorp.go) defines `GorpPlugin`, which is a plugin that does a couple things:
+[`app/controllers/gorp.go`](https://github.com/revel/samples/blob/master/booking/app/controllers/gorp.go) defines `GorpPlugin`, which is a plugin that does a few things:
```

Comment:
It doesn't really implement anything and it's also not a plugin. It's a controller with functionality that can be embedded into other controllers. Plugin is not a concept in Revel, so we shouldn't use it.